core/sta/ecc_self_tests.c

```diff
@@ -208,6 +373,8 @@ static TEE_Result invoke_command(void *pSessionContext __unused,
 	switch (nCommandID) {
 	case CMD_ECC_GEN_KEY_SELF_TESTS:
 		return ecc_generate_key_tests(pParams[0].value.a);
+	case CMD_ECC_DSA_TESTS:
+		return ecc_dsa_tests(pParams[0].value.a);
```

Comment:
I'm not a big fan of having tests in optee_os, they belong to optee_test IMO. The problem here is that you don't provide the client application that exercises the self-tests (or did I miss something?) so they are of no use to most people.